.babelrc

```diff
@@ -1,14 +1,22 @@
 {
```

Comment:
File is not changed - shouldn't be in diff

Reply:
rolled back